dashboard/index.html

```diff
@@ -0,0 +1,231 @@
+                                                    <td class="p-1 pt-3 border-b border-gray-600">{{ bot.drawdown }} {{ bot.uwdays }} d</td>
+                                                    <td class="p-1 pt-3 border-b border-gray-600"></td>
+                                                    <td class="p-1 pt-3 border-b border-gray-600"></td>
+                                                    <td class="p-1 pt-3 border-b border-gray-600" :class="classFromResult(bot.totalWorstCase)">{{ formatResult(bot.totalWorstCase) }}</td>
+                                                    <td class="p-1 pt-3 border-b border-gray-600">R={{ bot.risk_reference }}</td>
+                                                </tr>
+                                                <template v-for="position in bot.positions" :key="position.id">
+                                                    <tr class="divide-x divide-gray-400" :class="classFromPosition(position)"> 
+                                                        <td class="p-1 border-b border-gray-400">{{ position.id }}</td>
+                                                        <td class="p-1 border-b text-center uppercase">{{ position.status }}</td>
+                                                        <td class="p-1 border-b">{{ formatTime(position.signal_tstamp) }}</td>
+                                                        <td class="p-1 border-b">{{ position.amount }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.wanted_entry) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.initial_stop) }}</td>
+                                                        <td class="p-1 border-b">{{ formatTime(position.entry_tstamp) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.filled_entry) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.currentStop) }}</td>
+                                                        <td class="p-1 border-b" :class="classFromResult(position.worstCase)">{{ formatResult(position.worstCase) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.initialRisk) }}</td>
+                                                    </tr>
+                                                </template>
+                                            </template>
+                                        </tbody>    
+                                    </table>
+                                </div>
+                            </div>
+                        </template>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </div>
+
+
+    <script>
+        const Dashboard = {
```

Comment:
do we need this as part of the html or can we extract it into a .js file?

Reply:
We can extract it into a JS file. Tried to keep it simple so the dynamic template on the top can be developed alongside the logic that is used to render it - Basically "SFC / Single Page Components" https://vuejs.org/v2/guide/single-file-components.html without the need of having a build system :)